epispot/__init__.py

```diff
@@ -1,28 +1,173 @@
+To access some of epispot's metadata from the package itself and double-check your installation, run:
+```python
+>>> import epispot as epi
+>>> print(epispot.version)
+>>> '2.x.x'
+
+>>> print(epispot.stable)
+>>> True
+
+>>> print(epispot.source)
+>>> 'https://www.github.com/epispot/epispot'
+
+>>> print(epispot.docs)
+>>> 'https://epispot.github.io/epispot'
+
+>>> print(epispot.issues)
+>>> 'https://www.github.com/epispot/epispot/issues'
+
+>>> help(epispot)
+>>> 'Returns this docstring ...'
+
+>>> help(epispot.some_module.some_function)
+>>> 'Returns function docstring ...'
+```
+
+## Getting Started
+Start with the tutorials in `tests/tutorials`
+(see [GitHub repo](https://www.github.com/epispot/epispot/tree/master/tests/tutorials)). Next, check out the
+documentation at https://epispot.github.io/epispot. You may also find
+it helpful to see the examples in the `tests/examples` folder.
+
+## Release Notes
+See [GitHub repo](https://www.github.com/epispot/epispot/releases) for more info.
+
+## Features
+- Compartmental models with the following compartments:
+    - Susceptible (traditional SIR)
+    - Infected (traditional SIR)
+    - Recovered (traditional SIR)
+    - Exposed
+    - Dead
+    - Critical
+    - Hospitalized
+- Custom-defined compartmental models
+- Graphing and visualization engine
+    - Plot model predictions interactively
+    - Compare different model predictions
+
+## A Note on Documentation
 Docstrings provide additional documentation on a certain function.
 They can be accessed via the built-in Python `help()` command.
-These strings are formatted in Github-flavored markdown.
+These strings are formatted in GitHub-flavored markdown.
 Additionally, all files will have a 'STRUCTURE' label.
+
+## Credits
+The epispot package is supported by the following contributors:
+ - Head of Software & Development: [@quantum9innovation](https://www.github.com/quantum9innovation)
+ - Head of Code Maintenance: [@Quantalabs](https://www.github.com/quantalabs)
+
+Thank you also to all [GitHub contributors](https://www.github.com/epispot/epispot/insights/contributors)
 """
 
+
+def _dependency_check():
+    """Checks dependencies"""
+    try:
+        import numpy
+    except ImportError:  # pragma: no cover
+        raise ImportError('''Epispot requires numpy for integrating models; install with either:
+                                 >>> pip install numpy  # pip
+                                 >>> conda install numpy  # anaconda''')
+    try:
+        import matplotlib
+    except ImportError:  # pragma: no cover
+        raise ImportError('''Epispot requires matplotlib for creating interactive plots of models; install with either:
+                                 >>> pip install matplotlib  # pip
+                                 >>> conda install matplotlib  # anaconda''')
+
+
 # imports
 import warnings
 import random
+
+
+# dependencies
+_dependency_check()  # check for uninstalled dependencies
```

Comment:
You shouldn't have the dependency check after running it.
```suggestion
_dependency_check()  # check for uninstalled dependencies
del _dependency_check()
```